### ISSUE TITLE
Stopped GoHANmem in middle of search, now won't display menu again

### ISSUE THREAD

**Issue by drummerix** [OneRice07/GoHANmem#16]
I started a long search and got inpatient.  I closed the game and rebooted the Vita.

Now when I press the buttons to go into GoHANmem from inside the game (this case Dragon Quest Heroes 2), it just freezes and don't display the menu.

Is there a way to reset GoHANmem to not be in middle of searching again.

I tried letting it sit for awhile, but it didn't seem to help.  It was searching a 2 byte for about 16000 codes and taking some time when I closed the game in the middle of it searching.

I tried deleting tmp folder, but no help.

**Comment by AppNasty** [comment]
yeah. searching for high numbers is faster. if you have 25 gold....wait
till you have over 100 pus. then earch. make sure it's an odd uncommon
number. 100 will be common in games memory but 137 won't be so it's faster.
I've also noticed in some games when it's going slow, if you hit home
button, then when screen zooms out, go back into game and sometimes it will
shoot through quickly.
